src/main/java/org/jitsi/jicofo/FocusManager.java

```diff
@@ -619,6 +618,8 @@ private void expireLoop()
                     // Loop over conferences
                     for (JitsiMeetConference conference : conferenceCopy)
                     {
+                        System.out.println("XPIRE: " + conference.getRoomName()
```

Comment:
Yea, left over, I've been adding a bunch of these then later deleting them in the later commits to learn a bit more about the reservation system